joplin/db/backups/default-58807f2ec735-2018-09-27-073204.psql

```diff
@@ -0,0 +1,5612 @@
+--
```

Comment:
This is the only DB backup we actually need, as it's the most recent.

1. The first backup was made from master before any changes
2. Then I made one after adding the new steps model without removing the old one and running the schema migrations for that.
3. Then I made another one after creating and running the data migrations to get the steps from the old model to the new one.
4. That's this one! It's from the DB once we removed the old steps model.